### PR TITLE
Speed_glance

### PR DESCRIPTION
**Pull request by dkulykovmirantis**

### CODE REVIEW

File: mk_verificator/tests/test_speed_glance.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    speed_download = int(config['size_image_mb']) / (end_time - start_time)
+
+    start_time = time.time()
+    with open(name_image, 'wb') as image_file:
+        for item in glance_client.images.data(image.id):
+            image_file.write(item)
+    end_time = time.time()
+
+    speed_upload = int(config['size_image_mb']) / (end_time - start_time)
```
Comment (msenin94) [inline mk_verificator/tests/test_speed_glance.py]: please, check that md5sum of uploaded image is the same with local

File: mk_verificator/tests/test_speed_glance.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
+@pytest.fixture
+def create_image():
+    config = utils.get_configuration(__file__)
+
+    line = 'dd if=/dev/zero of={} bs=1M count=' \
+           '{}'.format(name_image, config['size_image_mb'])
+
+    os.system(line)
```
Comment (msenin94) [inline mk_verificator/tests/test_speed_glance.py]: that's good approach, but obviously for subprocess module is used

File: mk_verificator/tests/test_speed_glance.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    with open(name_image, 'wb') as image_file:
+        for item in glance_client.images.data(image.id):
+            image_file.write(item)
+    end_time = time.time()
+
+    speed_upload = int(config['size_image_mb']) / (end_time - start_time)
+
+    glance_client.images.delete(image.id)
+
+    print "++++++++++++++++++++++++++++++++++++++++"
```
Comment (msenin94) [inline mk_verificator/tests/test_speed_glance.py]: instead of print let's use fixtures with results

**Comment by dkulykovmirantis** [comment]
Waiting for an answer for benchmark part, Maybe it will be remove.

**Pull request closed without merging**